_opcodes/op175-bgee.html

```diff
@@ -22,3 +22,10 @@
 <br/>9   Kit.ids (EE only)
 <br/>
 <br/>This effect sets stat #58 (HELD) to 1. It also sets the STATE_HELPLESS flag.
+
+{% capture note %}
+At least in EEs, this opcode automatically provides the string specified by <code>STRREF_EFFECT_HOLD</code> in <code>ENGINEST.2DA</code>.
```

Comment:
What do you mean by "provide", print?

Reply:
> What do you mean by "provide", print?

Yes, it prints that string in the combat log.